content/applications/hr/referrals.rst

```diff
@@ -776,3 +458,8 @@ spreadsheet from the drop-down menu, or select :guilabel:`New Spreadsheet`. Clic
 .. note::
    The spreadsheet is stored in the *Documents* application. This application needs to be installed
    in order to use the :guilabel:`Insert in Spreadsheet` option.
+
```

Comment:
Suggestion to use a seealso to link to sub-docs as the toctree style isnt great for a single sub doc,
if implemented, please remove the `:show-toc:` meta tag from referrals.rst
```suggestion

.. seealso::
   :doc:`referrals/share_jobs`
   
```